README.md

```diff
@@ -171,6 +171,22 @@ While adding packages, please check if any of them are (now) available in Debian
 
 After the depdendencies are built, you can copy over `foreman`, `foreman-proxy` and `foreman-installer` from another release directory.
 
+## HOWTO: remove a Debian release
+
+First make sure the removal is announced, usually done with an RFC and a release note.
+For this a newer Debian/Ubuntu release needs to be supported together with the old release.
+This gives users a chance to upgrade their distribution separate from the Foreman version.
+
+1. Update [jenkins-jobs](https://github.com/theforeman/jenkins-jobs)
+  * Update the `operating_system` for dependency building in [`theforeman.org/pipelines/lib/packaging.groovy`](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/lib/packaging.groovy) if it's the oldest release
+  * Update `foreman_debian_releases` in [`theforeman.org/pipelines/vars/foreman/nightly.groovy`](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/vars/foreman/nightly.groovy)
+2. Drop it from foreman-packaging's `deb/develop` branch
+  * Drop the packaging definitions: `git rm -r {debian,dependencies}/$SUITE`
+  * Update any remaining entries in the repository: `$EDITOR $(git grep -l $SUITE)`
+3. Remove nightly packages
+  * `sudo rm -r /var/www/freight/apt/$SUITE/nightly && sudo -u freight /usr/bin/freight-cache -c /home/freight/freight.conf`
+  * `sudo rm -r /var/www/freightstage/apt/$SUITE/theforeman-nightly && sudo -u freightstage /usr/bin/freight-cache -c /home/freightstage/freight.conf`
```

Comment:
I'd not call freight here and just wait, but technically correct

Reply:
Because it takes time and there is nothing pressing to wait for it

Reply:
Running it in the background isn't that big of a deal